FPM.ftd

```diff
@@ -5,9 +5,14 @@ zip: github.com/fifthtry/ftd.dev/archive/refs/heads/main.zip
 language: en
 
 -- fpm.dependency: fifthtry.github.io/doc-site as ft
+
+-- fpm.dependency: fifthtry.github.io/brand-switcher as bs
```

Comment:
Remove this and also check if it's imported anywhere. `brand-switcher` is now known as `product-switcher`